src/client/components/icons/Info.tsx

```diff
@@ -3,7 +3,12 @@ import { colorsV3 } from '@hedviginsurance/brand'
 import { IconRoot, IconRootProps } from './IconRoot'
 
 export const InfoIcon: React.FC<IconRootProps> = (props) => (
-  <IconRoot viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" {...props}>
+  <IconRoot
+    viewBox="0 0 16 16"
+    xmlns="http://www.w3.org/2000/svg"
+    color={colorsV3.gray900}
+    {...props}
+  >
```

Comment:
What do you think about this solution? It would set a default value that can still be overridden if we use the icon on a dark background for example.

```
export const InfoIcon: React.FC<IconRootProps> = ({
  color = colorsV3.gray900,
  ...props
}) => (
  <IconRoot
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
    color={color}
    {...props}
  >
```

Reply:
Hmm you are probably right that "currentColor" should be the default (IconRoot takes care of that).

I just suggest this to keep the behavior from before. But it's probably better to pass the color in the context where we use the icon :)

Reply:
Yeah, at first I tried this one as it's kind of the standard way do deal with `defaultProps` with Typescript. Also works, I guess I didn't move on with it because I still would need to explicitly set `color` props to `IconRoot` so I thought it would be simpler just added it and still spread `props`. This solution shines when we have scenarios like the following, where we guarantee default values but don't intent to spread the props:

```
type LabelProps = { text?: string }

function Label({ text = 'default' }: LabelProps) {
  return <span>{text}</span>
}
```

However, I'm totally fine to change it to the proposed way :)

Reply:
Gonna remove `color` definition from `InfoIcon` then :)

Reply:
One thing tho, is that if we remove `color={colorsV3.gray900}` fron `InfoIcon` we would need to check for places where we're using `InfoIcon` without setting a `color` and end up setting it. Like [here](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/pages/OfferNew/Checkout/CreditCheckInfo.tsx#L41) for example, right?!

Reply:
Oh yeah, I think that's the way to go!

Explicitly pass color to `InfoIcon` wherever it's implemented :)

Reply:
Sounds good, it's never bad to set the color explicitly :) But my thought with using `currentColor` is that you don't have to since it inherits from it's parent that has color. Or was ther an issue with the color in any other browser? 

https://user-images.githubusercontent.com/6661511/139806626-9b518c2b-6d9a-4cd1-9fef-9275841b78e8.mov

.

Reply:
I agree. We should only override the and give an explicit color if it differs from the color of the text :)